components/profile_picture/profile_picture.tsx

```diff
@@ -94,14 +97,18 @@ export default class ProfilePicture extends React.PureComponent<Props> {
             );
         }
         return (
-            <span className='status-wrapper'>
+            <span className={`status-wrapper style--none ${this.props.wrapperClass}`}>
                 <span className={profileIconClass}>
                     <Avatar
                         size={this.props.size}
                         url={this.props.src}
                     />
                 </span>
-                <StatusIcon status={this.props.status}/>
+                {this.props.newStatusIcon ?
+                    <StatusIconNew
+                        className={this.props.statusClass}
+                        status={this.props.status}
+                    /> : <StatusIcon status={this.props.status}/>}
```

Comment:
Could you please wrap the first expression from the ternary in paranthese? Thanks!  🙇🏻